From fa4e58cadc3ea6f95c0756016f16818cc420cecc Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Thu, 3 Jan 2013 14:40:26 +0100
Subject: Also search resources in /var/lib/ultracopier

Add /var/lib/ultracopier to the list of places to look for resources.

Origin: vendor
Forwarded: no
Last-Update: 2013-01-03
---
 ResourcesManager.cpp |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/ResourcesManager.cpp b/ResourcesManager.cpp
index 008bf50..3839855 100644
--- a/ResourcesManager.cpp
+++ b/ResourcesManager.cpp
@@ -48,9 +48,12 @@ ResourcesManager::ResourcesManager()
 			#define EXTRA_HOME_PATH "/.config/Ultracopier/"
 		#endif
 		#ifdef Q_OS_LINUX
-			QDir linuxDir("/usr/share/ultracopier/");
-			if(linuxDir.exists())
-				searchPath<<"/usr/share/ultracopier/";
+			QDir linuxArchIndepDir("/usr/share/ultracopier/");
+			if(linuxArchIndepDir.exists())
+				searchPath<<ResourcesManager::AddSlashIfNeeded(linuxArchIndepDir.absolutePath());
+			QDir linuxPluginsDir("/usr/lib/ultracopier/");
+			if(linuxPluginsDir.exists())
+				searchPath<<ResourcesManager::AddSlashIfNeeded(linuxPluginsDir.absolutePath());
 		#endif
 		//load the user path but only if exists and writable
 		QDir dir(QFSFileEngine::homePath()+EXTRA_HOME_PATH);
